server/sonar-web/src/main/js/app/components/nav/settings/SettingsNav.tsx

```diff
@@ -82,118 +83,135 @@ export default class SettingsNav extends React.PureComponent<Props> {
 
   renderConfigurationTab() {
     const { organizationsEnabled } = this.props;
-    const configurationClassNames = classNames('dropdown-toggle', {
-      active:
-        !this.isSecurityActive() &&
-        !this.isProjectsActive() &&
-        !this.isSystemActive() &&
-        !this.isSomethingActive(['/admin/extension/license/support']) &&
-        !this.isMarketplace()
-    });
     const extensionsWithoutSupport = this.props.extensions.filter(
       extension => extension.key !== 'license/support'
     );
     return (
-      <li className="dropdown">
-        <a
-          className={configurationClassNames}
-          data-toggle="dropdown"
-          href="#"
-          id="settings-navigation-configuration">
-          {translate('sidebar.project_settings')} <i className="icon-dropdown" />
-        </a>
-        <ul className="dropdown-menu">
-          <li>
-            <IndexLink activeClassName="active" to="/admin/settings">
-              {translate('settings.page')}
-            </IndexLink>
-          </li>
-          <li>
-            <IndexLink activeClassName="active" to="/admin/settings/encryption">
-              {translate('property.category.security.encryption')}
-            </IndexLink>
+      <Dropdown>
+        {({ onToggleClick, open }) => (
+          <li className={classNames('dropdown', { open })}>
+            <a
+              className={classNames('dropdown-toggle', {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 3 times. ([typescript:S1192](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1192))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3154&issues=AWIkb2CXH1UYAK0FSViZ&open=AWIkb2CXH1UYAK0FSViZ)

[ISSUE_KEY]: AWIkb2CXH1UYAK0FSViZ